analysis/README.md

```diff
@@ -61,7 +70,20 @@ This installs all binaries to run the analysis. Java 11 is recommended, while Ja
 
 To run it you can call `ccsh`
 
-## Installation
+## Installation via Docker
+
+You can use this via docker compose or as a standalone container. This section will deal with how to use the analysis as a standalone container. For information on how to use this with docker compose, please check out [Docker Getting Started]({{site.baseurl}}{% link _docs/01-04-docker-containers.md %})
+We assume that you already installed docker, if not, you have to do that before!
+
+To containerize the analysis, please follow the below listed steps.
+
+-   Navigate into the directory you want to analyse with CodeCharta. There are multiple ways to use the docker image:
+    1. Start the docker container and a bash shell in it with `docker run -it -v $(pwd):$(pwd) -w $(pwd) codecharta/codecharta-analysis bash` (for that to work you have to make sure bash is added to your PATH). This runs the image and mounts the current directory and sets it as the working directory in the image. You can now use the CodeCharta shell or any other of the installed tools via the command line.
```

Comment:
By the stated docker command, the bash command is executed with in a running instance of the image (called container). So there is no need that the users have bash installed on their local system (including setting path variables).

Furthermore, I would add a sentence like this: You can use all the commands availalable in the container <Link to Docker Image or List of commands>. Do we have a list of available commands documented somewhere?

Reply:
Thanks for the info!

Right now I did not find a list with all available commands. I can add a list with all tools and how you can find out more about them (like the help command for each major tool/command)